pom.xml

```diff
@@ -105,6 +105,16 @@
               <goal>shade</goal>
             </goals>
             <configuration>
+	      <filters>
+		<filter>
+		  <artifact>*.*</artifact>
+		  <excludes>
```

Comment:
These lines should be further indented.

Reply:
Oh, oops, I think this is tabs vs spaces.